_docs/concepts/traffic-management/request-routing.md

```diff
@@ -1,28 +1,29 @@
 ---
 title: Request Routing
-overview: Describes how services are modeled within the Istio mesh, the notion of multiple versions of a service, and the communication model between services.
+overview: Describes how requests are routed between services in an Istio service mesh, including the Istio service model and service versions.
```

Comment:
Let's not mention service model here. How about just

Describes how requests are routed between services in an Istio service mesh.